cam_node:
    ros__parameters:
        fps: 20
        camera_id: 0
        publish_compressed: true
        publish_uncompressed: true
        apply_image_processing: false
        roi: [0, 0, -1, -1] # ROI as [start_row, start_col, end_row, end_col]
        adaptive_frame_rate: true
        compression_quality: 90 # From 0 (worst) to 100 (best)
        transform:
            translation:
                x: 0.0
                y: 0.0
                z: 1.0
            rotation:
                x: 0.0
                y: 0.0
                z: 0.0
                w: 1.0
